pkg/asset/manifests/proxy.go

```diff
@@ -62,10 +62,27 @@ func (p *Proxy) Generate(dependencies asset.Parents) error {
 	}
 
 	if installConfig.Config.Proxy != nil {
+		noProxy := installConfig.Config.Proxy.NoProxy
```

Comment:
Validation should be done in the types validation.
https://github.com/openshift/installer/blob/2952715dcf96a2529047aafd5f047db213c43474/pkg/types/validation/installconfig.go#L48